test/k9.test.ts

```diff
@@ -534,6 +536,84 @@ describe('K9KeyPolicy', () => {
 
 });
 
+describe('DynamoDBResourcePolicy', () => {
```

Comment:
nit: break the tests into files to match src.

Reply:
yes, might do that.